plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePageSource.java

```diff
@@ -337,6 +338,9 @@ private static Optional<Function<Block, Block>> createCoercer(TypeManager typeMa
         if (fromType instanceof DecimalType && toType == REAL) {
             return Optional.of(createDecimalToRealCoercer((DecimalType) fromType));
         }
+        if (fromType instanceof DecimalType && toType instanceof VarcharType) {
+            return Optional.of(createDecimalToVarcharCoercer((DecimalType) fromType));
```

Comment:
The commit title says "to UNBOUNDED VARCHAR", but the code seems to say "to varchar" (any)
What am i missing?

Reply:
My assumption was that varchar without adjective means unbounded one.  It was obviously wrong. Anyway I squished commits and removed adjectives so it should be fine now

Reply:
Thanks. Let's update PR title as well

Reply:
> My assumption was that varchar without adjective means unbounded one

in SQL, `varchar` denotes unbounded varchar type.